i18n/zh-hk/messages.zh-hk.xlf

```diff
@@ -5070,8 +5070,7 @@
       app.
     </source>
         <target state="new">
-      Global settings are stored in config map, so all of them are applied for every instance of the
-      app.
```

Comment:
Could you help to check why this is in a new line? @shu-mutou

Reply:
You can find original context from following `context` tag, i.e. line 5077.
It points https://github.com/kubernetes/dashboard/blob/master/src/app/frontend/settings/global/template.html#L23-L26

Reply:
ahh, I checked it on the UI page, they are placed in the `p` label on the same line, and it looks like an optional line break. In any case, he will not cause problems.